src/graphql/Question.ts

```diff
@@ -0,0 +1,58 @@
+import { objectType, intArg, stringArg, arg, core, nonNull } from "nexus";
+
+export const Question = objectType({
+  name: "Question", // <- Name of your type
+  definition(t) {
+    t.int("id"); // <- Field named `id` of type `Int`
+    t.list.field("translation", {
+      type: QuestionTranslation,
+      resolve(root, _args, ctx) {
+        return ctx.db.questionTranslation.findMany({
+          where: { questionId: { equals: root.id } },
+        });
+      },
+    });
+  },
+});
+
+export const QuestionTranslation = objectType({
```

Comment:
This type sems to be a duplicate of the one in `QuestionTranslation.ts`.
Can you move this type inthe other file ?